pkg/vm/engine/disttae/table.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package disttae
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/matrixorigin/matrixone/pkg/container/batch"
+	"github.com/matrixorigin/matrixone/pkg/container/vector"
+	"github.com/matrixorigin/matrixone/pkg/pb/plan"
+	"github.com/matrixorigin/matrixone/pkg/vm/engine"
+)
+
+var _ engine.Relation = new(table)
+
+func (tbl *table) Rows() int64 {
+	return 0
+}
+
+func (tbl *table) Size(name string) int64 {
+	return 0
+}
+
+func (tbl *table) Ranges(ctx context.Context) ([][]byte, error) {
+	return nil, nil
+}
+
+func (tbl *table) TableDefs(ctx context.Context) ([]engine.TableDef, error) {
+	return nil, nil
+}
+
+func (tbl *table) GetPrimaryKeys(ctx context.Context) ([]*engine.Attribute, error) {
+	return nil, nil
+}
+
+func (tbl *table) GetHideKeys(ctx context.Context) ([]*engine.Attribute, error) {
```

Comment:
HiddenKeys

Reply:
> HiddenKeys


I don't even think this interface should be needed, with the TableDefs() interface you can fetch all the information of the table. This type of interface my plan is to remove after 0.6 is ready to run.